app.yaml

```diff
@@ -2,9 +2,6 @@ runtime: python27
 api_version: 1
```

Comment:
Per our discussion, let's just have one app.yaml file in the codebase so that there's a single source of truth.

@vojtechjelinek FYI we discussed having a template file from which to generate dev/prod, but then we figured that it would be better to just have the "dev" version in the codebase (as we do now) and then do the changes needed to transform it to the "prod" version (like we do with other files/folders). This means that we only have one indirectly-generated file rather than two (which is better, since indirect pipelines like this often lead to errors).

Reply:
Done

Reply:
@seanlip, should this file still be in the codebase?

Reply:
No, I don't think so.